src/components/timeline/Timeline.ts

```diff
@@ -79,6 +79,8 @@ export namespace Timeline {
           this.streamId
         ) {
           flag = true;
+        } else if(name === "tapeEvents"){
```

Comment:
@elenagiere does this condition need anything else?

Reply:
`TapeEvents` got renamed to `timelineEvents`. Take a look at line 42:
`@property({ type: Array }) timelineEvents: TimelineEvent[] = [];`

We also can write this more cleaner. We should have super.updated(changedProperties); on line 70.

Reply:
I believe the updated function could be cleaned up like so
```
    updated(changedProperties: PropertyValues) {
      super.updated(changedProperties);

      if (
        (changedProperties.has("streamId") && this.streamId) ||
        changedProperties.has("timelineEvents") ||
        (changedProperties.has("filter") && this.streamId)
      ) {
        this.timelineEvents = [];
        this.requestUpdate();
        this.subscribeToStream();
      }
    }
    ```

Reply:
So, why are we adding this? If we add it correctly using the appropriate new property name, it ends up in an infinite loop. I'm guessing it wasn't working from Profile, maybe because you were passing in the old property name?

Reply:
Understood. Disregard this PR.